### ISSUE TITLE
Create Login Screen

### ISSUE THREAD

**Issue by bw-davis** [bw-davis/hansel#7]
- [ ] create the view controller
- [ ] write the appropriate code ([docs](https://firebase.google.com/docs/auth/ios/manage-users))

**Comment by natsn** [comment]
https://www.firebase.com/docs/ios/guide/login/google.html

**Comment by natsn** [comment]
https://github.com/firebase/login-demo-ios

**Comment by nqthqn** [comment]
@bw-davis Currently our database is read/write accessible without authentication. Once you have this implemented I will turn that off. Also, could you show me how you do this? making the view controller and all? Maybe on Sunday

**Comment by bw-davis** [comment]
@natsn Yeah. We can study and then we can create the login screen together.
